eng/testing/performance/performance-setup.sh

```diff
@@ -279,14 +279,12 @@ if [[ "$internal" == true ]]; then
     elif [[ "$logical_machine" == "perfampere" ]]; then
         queue=Ubuntu.2004.Arm64.Perf
     elif [[ "$logical_machine" == "cloudvm" ]]; then
-        queue=Ubuntu.1804.Amd64
+        queue=Ubuntu.2204.Amd64
     elif [[ "$architecture" == "arm64" ]]; then
         queue=Ubuntu.1804.Arm64.Perf
     else
         if [[ "$logical_machine" == "perfowl" ]]; then
-            queue=Ubuntu.1804.Amd64.Owl.Perf
-        elif [[ "$logical_machine" == "perftiger_crossgen" ]]; then
```

Comment:
`perftiger_crossgen` requires using a version of lttng that's not available on 22